src/components/TokenBalance.tsx

```diff
@@ -221,6 +244,21 @@ const styles = StyleSheet.create({
   totalAssets: {
     ...fontStyles.regular600,
     color: Colors.gray5,
+    marginRight: 4,
+  },
+  totalAssetsInfoContainer: {
+    position: 'absolute',
+    top: 24,
```

Comment:
Related to my other comment, does this ensure the message appears below the info icon? Because, if somehow this is shown on top of the icon, the user won't be able to dismiss it.